pacfetcher_test.go

```diff
@@ -129,3 +129,24 @@ func TestPacFromFilesystem(t *testing.T) {
 	assert.Equal(t, content, pf.download())
 	assert.True(t, pf.isConnected())
 }
+
+func TestDecodeDataURL_Base64(t *testing.T) {
+	uri := "data:application/x-ns-proxy-autoconfig;base64,ZnVuY3Rpb24gRmluZFByb3h5Rm9yVVJMKHVybCwgaG9zdCkgewogIHJldHVybiAiUFJPWFkgcHJveHk6ODA4MCI7Cn0K"
+	want := []byte("function FindProxyForURL(url, host) {\n  return \"PROXY proxy:8080\";\n}\n")
+	pf := newPACFetcher(uri)
+	assert.Equal(t, want, pf.download())
```

Comment:
i find the output of assert.Equal() to be really hard to read when the want/got values are of type `[]byte` (rather than `string) and the assertion fails. can you convert the arguments to strings before doing the assertion?

```suggestion
	want := "function FindProxyForURL(url, host) {\n  return \"PROXY proxy:8080\";\n}\n"
	pf := newPACFetcher(uri)
	assert.Equal(t, want, string(pf.download()))
```